rpc_server.go

```diff
@@ -156,8 +156,11 @@ type MixFractionObject struct {
 // But changes to the mix settings need to be kept separate from LanceroSource.distrubuteData,
 // which is part of the data-*production* step, not the data-processing step.
 func (s *SourceControl) ConfigureMixFraction(mfo *MixFractionObject, reply *bool) error {
-	err := s.ActiveSource.ConfigureMixFraction(mfo)
+	currentMix, err := s.ActiveSource.ConfigureMixFraction(mfo)
 	*reply = (err == nil)
+	if err == nil {
```

Comment:
make broadcast unconditional?

Reply:
Sure. I'll take care of that and then finish this PR. Thanks for reviewing.